doc/sources/installation/installation-windows.rst

```diff
@@ -230,6 +239,36 @@ kivy with git rather than a wheel there are some additional steps:
      export USE_GSTREAMER=1
 
    These variables must be set everytime you recompile kivy.
+   
+   **MSVC:**
```

Comment:
I don't have the time right now to re-verify this locally, but I can say unequivocally that this did not work for me yesterday. Without a doubt, the CD + INCLUDE is required for the venv; that's what tells it to look for the headers locally instead of system-wide. Setup.py is LBYL, so otherwise it ignores SDL. See #4941.